azext_iot/tests/central/test_iot_central_devices_int.py

```diff
@@ -87,17 +73,13 @@ def test_central_device_twin_show_success(self):
 
         self.cmd(
             command,
-            checks=[self.check("deviceId", device_id)],
-        )
-
-        command = "iot central device twin show --app-id {} --device-id {}".format(
-            APP_ID, device_id
+            checks=[
+                self.check("deviceId", device_id),
+                self.check("tags", {}),
```

Comment:
Make tags a generic dict (so you populate the field with something first) and check that.

Reply:
in Central all twins have empty object tags. This test also verify that no external change to twin has been made. Will keep as is.